Scripts/script-generateSummaryReport.yml

```diff
@@ -0,0 +1,23 @@
+commonfields:
+  id: GenerateInvestigationSummaryReport
+  version: -1
+fromversion: 3.5.0
+name: GenerateInvestigationSummaryReport
+script: 'return executeCommand("generateSummaryReport", {name: args.name, type: args.type,
+  id: args.incidentId})'
+type: javascript
+tags:
+- post-processing
+enabled: true
+args:
+- name: name
+  default: true
+  description: Report name to generate (should correspond with report type)
+  defaultValue: Investigation Summary
+- name: type
+  description: Report type to generate (should correspond with report name)
+  defaultValue: pdf
+- name: incidentId
```

Comment:
do we really need the incidentId argument for post-processing?

Reply:
No, but this way this script can be used without pre processing as well